### PR TITLE
Minor code cleanup and dependency updates

### PR DESCRIPTION
**Pull request by ngerakines**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,16 +8,16 @@ repository = "https://github.com/yeastplume/rust-sssmc39"
...
-failure = "0.1"
-failure_derive = "0.1"
-rand = "0.6"
-hmac = "0.7"
-sha2 = "0.8"
-ring = { version = "0.16", optional = true }
-digest = { version = "0.8", optional = true }
-pbkdf2 = { version = "0.3", optional = true, default-features = false }
+lazy_static = "1.4.0"
+bitvec = "1.0.1"
```
Comment (wigy-opensource-developer) [inline Cargo.toml:12]: Pretty interesting that all the breaking changes in bitvec usage has left this codebase intact.

**Review by wigy-opensource-developer — COMMENTED** [review]
Nice `cargo clippy` and `cargo update` run.

**Comment by wigy-opensource-developer** [comment]
@yeastplume Are you willing to merge these updates? They seem to be quite useful to keep the library alive.

**Comment by yeastplume** [comment]
Sure, thanks for looking into this!

**Pull request merged**